Makefile

```diff
@@ -131,10 +131,11 @@ devtools-stamp: tools
 	go get github.com/tendermint/lint/golint
 	touch $@
 
-get_vendor_deps: tools
+vendor-deps: tools
```

Comment:
Why are we renaming this? I think it would be better to maintain consistency (for past scripts and with Tendermint) at this point.

Reply:
Because the target now creates a `vendor-deps` file. I thought `get_vendor_deps` was really a bad name for it (I actually always thought so). Happy to rename it back though.

Reply:
I don't have a preference either way, but I think @greg-szabo had asked for consistency for tooling.

Reply:
So can we not willy nilly rename the makefile stuff? If we want to change the makefile naming that should be done in 1 large PR to be consistent and all the stakeholders (i.e. @cosmos/sre) need to be brought in on the discussion.

FWIW I'm not crazy about the naming here either.

Reply:
I don't mind renaming as long as it also gets renamed in the CI config file.

Reply:
What Mircea and Jack said. Not crazy about current naming conventions but they seeped into everything. (Just look at some of the faucet code or the telegram bot.) I don't mind accepting this new naming as long as CircleCI is also updated _and_ if we (eventually) come up with how this should look like so it's not nilly-willy. :)

The `cat vendor-deps` part however won't work on Windows. Please use `$(file < vendor-deps)` instead. (And test it, this is off the top of my head.)

Reply:
$(file <) requires Make >= 4.2